--- allocate.c1	2018-10-23 14:59:59.505380955 +0800
+++ allocate.c	2018-10-23 15:00:43.705029363 +0800
@@ -307,7 +307,7 @@
 	bg->first_free_block = 0;
 	bg->free_inodes = info.inodes_per_group;
 	bg->first_free_inode = 1;
-	bg->flags = EXT4_BG_INODE_UNINIT;
+	bg->flags = EXT4_BG_INODE_UNINIT|EXT4_BG_INODE_ZEROED;
 
 	if (reserve_blocks(bg, bg->first_free_block, bg->header_blocks) < 0)
 		error("failed to reserve %u blocks in block group %u\n", bg->header_blocks, i);
